src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItem.jsx

```diff
@@ -125,6 +125,12 @@ export const formatSeries = (selectedItems, cardJson) => {
   return series;
 };
 
+export const formatDataItemsForDropdown = (dataItems) =>
```

Comment:
Does it have to be exported?

Reply:
Oh my intentions were to add a test case, but I forgot. Will add into the next iteration